<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">GameWidget#GameWidget{
	background:black
}</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="GameScene" name="gameScene" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>1620</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:grey</string>
    </property>
   </widget>
   <widget class="QLabel" name="introduceLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>300</width>
      <height>260</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 170, 255);
border-width: 1px;
border-style: solid;
border-color: white;
color:black;</string>
    </property>
    <property name="text">
     <string>TEST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(170, 255, 255);
border-width: 1px;
border-style: solid;
border-color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>Frame : 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="playerCountLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(241, 241, 241)</string>
    </property>
    <property name="text">
     <string>PlayerCount</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MusicPlayModule" name="musicPlayModule" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 170, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="message3Label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>800</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="message2Label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>800</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="message1Label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>800</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>300</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(206, 206, 206);
border-width: 1px;
border-style: solid;
border-color: gray;</string>
    </property>
    <widget class="QLabel" name="sublabel8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sublabel7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sublabel6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sublabel5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sublabel4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sublabel3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sublabel2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sublabel1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:2px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2{
	background: white;
	border-width: 1px;
	border-style: solid;
	border-color: rgb(255, 170, 0);
}</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>玩   家：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>生命值：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>武   器：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="playerNameLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>电脑</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="playerHealthLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100/100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="firearmsWeaponLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>无</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>淘   汰：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="disuseLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>技   能：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="skillLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>230</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 170, 255);
color:white;</string>
    </property>
    <property name="text">
     <string>剩余时间：500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="SummaryWidgetModule" name="summaryWidgetModule" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>1100</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="giftLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>700</y>
      <width>300</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 255, 255)</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameScene</class>
   <extends>QWidget</extends>
   <header>gamescene.h</header>
  </customwidget>
  <customwidget>
   <class>MusicPlayModule</class>
   <extends>QWidget</extends>
   <header>musicplaymodule.h</header>
  </customwidget>
  <customwidget>
   <class>SummaryWidgetModule</class>
   <extends>QWidget</extends>
   <header>summarywidgetmodule.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
